### PR TITLE
refactor(dms/rocketmq_topic): fix dms rocketmq topic lint issues

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix dms rocketmq topic lint issues
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix dms rocketmq topic lint issues
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms/' TESTARGS='-run TestAccDmsRocketMQTopic_basic' 
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDmsRocketMQTopic_basic -timeout 360m -parallel 4 
=== RUN   TestAccDmsRocketMQTopic_basic 
=== PAUSE TestAccDmsRocketMQTopic_basic
=== CONT  TestAccDmsRocketMQTopic_basic
--- PASS: TestAccDmsRocketMQTopic_basic (717.71s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       717.768s
```

### CODE REVIEW

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go [hunk 1]
```diff
@@ -160,7 +161,7 @@ func resourceDmsRocketMQTopicCreate(ctx context.Context, d *schema.ResourceData,
 	return resourceDmsRocketMQTopicUpdate(ctx, d, meta)
 }
 
-func buildCreateRocketmqTopicBodyParams(d *schema.ResourceData, config *config.Config) map[string]interface{} {
+func buildCreateRocketmqTopicBodyParams(d *schema.ResourceData, _ *config.Config) map[string]interface{} {
```
Comment (Lance52259) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go]: If the config is not used, we should remove it from the argument list.

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go [hunk 2]
```diff
@@ -91,7 +92,7 @@ func ResourceDmsRocketMQTopic() *schema.Resource {
 	}
 }
 
-func DmsRocketMQTopicBrokerRefSchema() *schema.Resource {
+func RocketMQTopicBrokerRefSchema() *schema.Resource {
```
Comment (chengxiangdong) [inline huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go]:
```suggestion
func rocketMQTopicBrokerRefSchema() *schema.Resource {
```
it seems it should be a private function

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go]: done

File: huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_topic.go]: done

**Pull request merged**